### PR TITLE
Added a new Rabbit Suit to Honkington!

### PR DESCRIPTION
**Pull request by nerkson**
[CLOTHING] [SPRITES]

## About the PR 
Added a Rabbit suit for sale from Geoff Honkington! In the box is a Rabbit Suit and Rabbit Hat, so mix and match to your hearts content!
Screenshot 2024-04-06 194219

## Why's this needed? 
I had wanted to add this in ahead of time for Easter but I was prevented from doing it by life. All the same it's a nice seasonal type costume to have! 

## Changelog 

```changelog
(u)Carthridge / Nerkson
(+)A new egg-celent costume has bounced into Geoff Honkington's store! Check his stock for more info!
```

### CODE REVIEW

**Review by 444Portal — CHANGES_REQUESTED** [review]
everything code-wise and sprites-wise looks good, could probably use a minor (+) changelog entry that tells you its in geoffs stock

**Comment by nerkson** [comment]
> everything code-wise and sprites-wise looks good, could probably use a minor (+) changelog entry that tells you its in geoffs stock

Updated, thank you!

**Review by 444Portal — APPROVED** [review]
ya looks good

**Review by amylizzle — APPROVED** [review]
This is great. Can't wait to be deeply unsettled by someone wearing this in a dark maint corridor.

File: code/obj/item/clothing/hats.dm [hunk 1]
```diff
@@ -2235,3 +2235,13 @@ ABSTRACT_TYPE(/obj/item/clothing/head/mushroomcap)
...
+
+/obj/item/clothing/head/rabbithat
+	name = "Rabbit Costume Hat"
+	desc = "You're gonna need a psych eval after wearing this. And a shower."
+	icon = 'icons/obj/clothing/item_hats.dmi'
+	wear_image_icon = 'icons/mob/clothing/head.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_headgear.dmi'
+	icon_state = "rabbithat"
+	item_state = "rabbithat"
+	seal_hair = 1
```
[the remark this answers is not in the document]
Comment (nerkson) [reply code/obj/item/clothing/hats.dm:2247]: Thank you! Made the change.

**Pull request merged**